src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3756,6 +3756,21 @@ void SpellMgr::LoadSpellInfoCorrections()
                 break;
             // ENDOF ISLE OF CONQUEST SPELLS
             //
+            case 18754: // Improved succubus (1 talent)
+                spellInfo->Effects[EFFECT_0].ApplyAuraName = SPELL_AURA_ADD_FLAT_MODIFIER;     // affects the duration of seduction, let's minimize affection 
```

Comment:
what are the current values ? the PR doesn't show the diff between old and new values for each modified spell